ui/admin/app/components/form/field/key-value-list/index.hbs

```diff
@@ -73,15 +78,17 @@
                 {{on 'input' (set-from-event this 'newOptionKey')}}
               />
             </B.Td>
-            <B.Td>
-              <Hds::Form::TextInput::Field
-                @value={{this.newOptionValue}}
-                @type='text'
-                disabled={{@disabled}}
-                aria-labelledby={{valueHeaderID}}
-                {{on 'input' (set-from-event this 'newOptionValue')}}
-              />
-            </B.Td>
+            {{#unless @hasNoValue}}
+              <B.Td>
+                <Hds::Form::TextInput::Field
+                  @value={{this.newOptionValue}}
```

Comment:
Do we need to handle the backing js file for not having a value now?

Reply:
I didn't change the js file because the `value` can be empty

Reply:
<img width="280" alt="Screenshot 2023-10-11 at 4 16 31 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/17d033e0-9b93-462c-be6f-1467cfc6c40d">

But open to changing this if this might cause confusion

Reply:
I personally think it's an easy to change to not send an empty string so would probably do that

Reply:
I was looking at an incorrect example with a `value` input field in the action params. If we don't send a `value` field to the `@addOption` action, we won't have the entry for value. So there is no need to change anything in the backing js file. 
<img width="542" alt="Screenshot 2023-10-12 at 2 19 11 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/926bb635-d757-45a5-be39-40ab29d6ec25">

Reply:
Right my point was we're sending a value field to the callback with just an empty string

Reply:
good point, I will clean this up in the following PR

Reply:
I think it's fine if it's been merged in, it's not that big of a deal. It's just an extra field that won't get used